web/content/docs/userguide/process-dependent-configuration/Heat_Transport_BHE_PipelineNetwork.pandoc

```diff
@@ -0,0 +1,278 @@
++++
+date = "2020-02-03T12:00:13+01:00"
+title = "Heat_Transport_BHE_PipeNetwork_feature"
+author = "Shuang Chen, Haibing Shao"
+weight = 1
+
+[menu]
+  [menu.userguide]
+    parent = "process-dependent-configuration"
++++
+
```

Comment:
Usually there is a  link here which points to the corresponding input files. Consider to add it.

Reply:
Added in 934ba904116c329b7b913b636acaadcc9e137b11.